pages/_app.js

```diff
@@ -1,10 +1,45 @@
 import GlobalStyle from "../styles";
+import { useState } from "react";
+import { initialGames } from "@/db/games/data";
```

Comment:
Again some mixing of import statements ;) Please stick to one syntax consistently.